internal/appSetup.go

```diff
@@ -90,61 +89,56 @@ func Unmount(mountPoint string) (err error) {
 }
 
 func InitializeApp(gen3FuseConfig *Gen3FuseConfig, manifestURL string, mountPoint string) {
-	app := cli.NewApp()
```

Comment:
why'd you decide to remove this construct? it appears to me to be crucial to the normal functioning of the program. when i perform a cat operation on a mounted file, i now see the output of the go program in addition to the contents of the file. i think this is because the lines you removed were creating an app shell which caught log output and deflected it away from stdout

Reply:
```Zakirs-MacBook-Pro:gen3-fuse zakirgowani$ cat mountpt/by-filename/s3test1.txt 
Inside ReadFile
get fbd5b74e-6789-4f42-b88f-f75e72777f5d with offset 0 size 211
211
this file lives in s3://devplanetv1-proj1-databucket-gen3 bucket, with corresponding records 
in the index_record, index_record_url, index_record_url_metadata, index_record_metadata, and index_record_ace tables```

Reply:
wow! thank you for catching that. I had removed the wrapper because the program was parsing the command line flags twice, once at the beginning for error checking to see if the required arguments were present, and then at this app step the flags are parsed again because of that cli wrapper. It seemed redundant to parse the flags twice, but I did not notice this issue with the redirect of the log output. I will make some changes and test again.

Reply:
I re-wrapped that bit of code, but not in that cli package struct, because that bit I believe was truly unnecessary. Seems to work now that that block is wrapped again. Please let me know that it works the same for you - thanks!